Core/include/Acts/EventData/SingleBoundTrackParameters.hpp

```diff
@@ -158,19 +176,18 @@ class SingleBoundTrackParameters {
   /// Access the time coordinate.
   Scalar time() const { return get<eBoundTime>(); }
 
-  /// Access the direction pseudo-rapidity.
-  Scalar eta() const { return -std::log(std::tan(get<eBoundTheta>() / 2)); }
-  /// Access the absolute transverse momentum.
-  Scalar pT() const {
-    return std::sin(get<eBoundTheta>()) / std::abs(get<eBoundQOverP>());
+  /// Direction unit three-vector, i.e. the normalized momentum three-vector.
+  Vector3D directionUnit() const {
+    return makeDirectionUnitFromPhiTheta(get<eBoundPhi>(), get<eBoundTheta>());
   }
-  /// Access the momentum three-vector.
-  Vector3D momentum() const {
-    auto mom =
-        makeDirectionUnitFromPhiTheta(get<eBoundPhi>(), get<eBoundTheta>());
-    mom *= std::abs(1 / get<eBoundQOverP>());
-    return mom;
+  /// Absolute transverse momentum.
+  Scalar transverseMomentum() const {
```

Comment:
Why rename this? `pT` is pretty much universal.

Reply:
The idea was to be consistent with the rest of the accessors. All of them are written out, e.g. `position` and `charge`. Thus, `transverseMomentum` and `absMomentum`. This is also used by `ActsFatras::Particle`.

Another option could be `.pt()` and `.p()`?

Reply:
Well if you write this out it should be `absoluteMomentum` as well. I think these names are all longer than they need to be.

Reply:
Rename to `absoluteMomentum` for consistency.